browser/net/url_context.cc

```diff
@@ -138,6 +138,9 @@ std::shared_ptr<brave::BraveRequestInfo> BraveRequestInfo::MakeCTX(
       brave_shields::GetBraveShieldsEnabled(map, ctx->tab_origin);
   ctx->allow_ads = brave_shields::GetAdControlType(map, ctx->tab_origin) ==
                    brave_shields::ControlType::ALLOW;
+  ctx->aggressive_blocking =
+      brave_shields::GetCosmeticFilteringControlType(map, ctx->tab_origin) ==
```

Comment:
this assignment is not really readable - I had to search through the source to understand that "aggressive" flag is stored as cosmetic filtering content setting, which we reuse here... So definitely needs a comment

Reply:
added as well